R/init.R

```diff
@@ -154,7 +154,13 @@ init <- function(data,
   teal.logger::log_system_info()
 
   # invoke lockfile creation
-  process <- teal_lockfile_invoke()
+  user_lockfile <- getOption("teal.renv.lockfile", "")
+  lockfile_process <- NULL
+  if (!identical(user_lockfile, "")) {
+    teal_lockfile_external()
+  } else {
+    lockfile_process <- teal_lockfile_process_invoke()
+  }
```

Comment:
This could also be wrapped into a high-level function

Reply:
added: 2a931b52c9ffec9ba92f29f05c76ab48619828e1